fuselage.py

```diff
@@ -367,13 +368,14 @@ def setup(self, Nmissions, **kwargs):
                 Vcabin >= Afuse * (lshell + 0.67 * lnose + 0.67 * Rfuse),
 
                 # Weight relations
-                Wapu == Wpay * fapu,
+                Wapu == Wpaymax * fapu,
                 Wdb == rhoskin * g * Vdb,
                 Winsul >= Wppinsul * ((1.1 * pi + 2 * thetadb) * Rfuse * lshell + 0.55 * (Snose + Sbulk)),
                 Wwindow >= Wpwindow * lshell,
-                Wpadd == Wpay * fpadd,
+                Wpadd == Wpaymax * fpadd,
+                # Two methods for determining seat weight (must be inequalities)
                 Wseat >= Wpseat * nseat,
-                Wseat >= fseat * Wpay,
+                Wseat >= fseat * Wpaymax,
 
```

Comment:
can we wrap everything in TCS that should be tight?

Reply:
K, I will add a few where I think it is necessary. Otherwise I don't think TCS is very useful; we have tons of untight constraints because we have multiple ways of sizing things.

Reply:
yeah i agree, that's what I meant by wrap things that should be tight

Reply:
Done.